src/components/CaptureDetailDialog.js

```diff
@@ -265,14 +299,87 @@ function CaptureDetailDialog(props) {
           )}
 
           <Typography variant="subtitle1">Other</Typography>
-          {allTags.length === 0 ? (
+          {otherTags.length + captureTags.length === 0 ? (
             <Typography variant="body1">---</Typography>
           ) : (
-            <div className={classes.chipRoot}>
-              {allTags.map((tag) => (
-                <Chip key={tag} label={tag} className={classes.chip} />
-              ))}
-            </div>
+            <>
+              <div className={classes.chipRoot}>
+                {otherTags.map((tag) => (
+                  <Chip key={tag} label={tag} className={classes.chip} />
+                ))}
+
+                {captureTags.map((tag) => (
+                  <Chip
+                    key={tag.tagName}
+                    label={tag.tagName}
+                    className={classes.chip}
+                    onDelete={
+                      hasApproveTreePermission !== undefined // TODO: delete, for testing purpose only
```

Comment:
As per the comment, should this be deleted?

Reply:
Yes, thank you, I have fixed it to correctly check for the APPROVE_TREE permission.